ansible_collections/arista/avd/roles/eos_designs/docs/tables/wan-path-groups.md

```diff
@@ -47,4 +51,16 @@
 
             # Optional, if not set, the path-group `name` is used as local.
             local: <str>
+
+        # Period between the transmission of consecutive keepalive messages, and failure threshold.
+        dps_keepalive_interval:
+
+          # Enable adaptive keepalive and feedback interval.
+          auto: <bool; default=False>
+
+          # Interval in milliseconds.
+          interval: <int; 50-60000>
+
+          # Failure threshold in number of intervals. Required when `interval` is set.
+          failure_threshold: <int; 2-100>
```

Comment:
In eos_designs we should implement the "best practice", which I assume is `auto` for this case?

Reply:
Nevermind. The best practice is to keep the defaults.